src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -60,20 +70,51 @@ class pt3(Macro):
     """Macro with a list of numbers as parameter: the type is a sequence of
     parameter types which is repeated. In this case it is a repetition of a 
     float so only one parameter is defined.
-    By default the repetition as a semantics of 'at least one'"""
+    By default the repetition as a semantics of 'at least one'
+    Usages, ex.:
+    pt3 [[1][34][15]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.